src/test/resources/uk.gov.homeoffice.digital.sas.person.bdd/person.feature

```diff
@@ -16,3 +31,106 @@ Feature: Person
       | field      | type    | expectation           |
```

Comment:
Align the separating pipes vertically for readability

Reply:
Done